Source/WebCore/dom/Text.idl

```diff
@@ -19,6 +19,7 @@
 
```

Comment:
No copyright change?

Reply:
I'll change it.

Reply:
addressed in https://github.com/WebKit/WebKit/pull/3099/commits/9de2b2c865e3b75444db4d1af2e7cf8226c10320